app/controllers/claims_controller.rb

```diff
@@ -12,6 +12,10 @@ class ClaimsController < BasePublicController
   before_action :clear_claim_session, only: [:new]
   before_action :prepend_view_path_for_journey
 
+  SLUG_TO_FORM = {
+    "current-school" => CurrentSchoolForm
+  }
```

Comment:
I think we might want to define this per journey, so that we can have different behaviours as necessary. I suggest we could move this to the relevant `SlugSequence` class, or the relevant `Journey` module. If we want to have a default behaviour, this could be defined in `Journeys::Base` perhaps.